src/server/models/users.model.js

```diff
@@ -8,6 +8,7 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
+  userId: String,
```

Comment:
I'm not sure if there's any flow in the product to get "gildong" as the user ID. We don't have a sign up flow, so all the info is coming from fb / li. So the "gildong" is merely coming from the mock data.

_id itself is present for every single object, right? So I think it's ok to duplicate them as user_id for user objects. I'm just not sure if that _id is persistent : if the mongodb crashes and we'll have to recover from a backup, would we still maintain that object _id? Things like that. 

Let's just use _id as the user_id for now, and in the mean time, let's find out how other systems are setting up user_ids in their applications.